### PR TITLE
STORY-17402: Remove hyphens in documentation ring group desinations

### PR DESCRIPTION
**Pull request by NickLindeberg**
[Jira Ticket](https://invoca.atlassian.net/browse/STORY-17402)
[Web PR](https://github.com/Invoca/web/pull/23045)

Removing hyphens from phone number in ring group destination response to be more consistent. 

## Checklist

- [X] Find the [Service owning team](https://docs.google.com/spreadsheets/d/1YF2wuepY5SZTpVEdT9gwhTPhpnDGZeVjywEwvbJS5TQ/edit#gid=0&fvid=1520238175) for these changes, and tag the team as "Reviewers" on this PR
- [ ] Test the documentation changes on readthedocs as a private branch
- [ ] If changing general content, have agreement on whether to apply to latest version or all versions (if all versions, provide links to the related PRs below)

### CODE REVIEW

File: source/api_documentation/network_integration/advertiser_campaigns/index.rst [hunk 1]
```diff
@@ -540,7 +540,7 @@ Parameter Details
 
   * - ring_group_destinations
     - Array of hashes
-    - A list of destination hashes for the RingGroup node. Each hash must contain a "ring_group_number" parameter at minimum. Other optional parameters are "description" (string), "destination_disabled" (boolean, default: false), and "distribution_weight" (integer, default: 1). "destination_disabled" when set to true, will prohibit the destination from being dialed as part of the RingGroup. "distribution_weight" may only be used when the distribution_method of the RingGroup node is set to "Weighted". For example, ring_group_destinations may have a value of [{"ring_group_number": "800-444-1111", "description": "Call Center 1", "distribution_weight": 2 }, {"ring_group_number": "800-444-2222", "description": "Call Center 2", "distribution_weight": 1 }].
```
Comment (danielmx-dev) [inline source/api_documentation/network_integration/advertiser_campaigns/index.rst:543]: Would it make sense to also add the `ring_group_country_code` field as well? I didn't update the docs when that field was added.

Comment (NickLindeberg) [reply source/api_documentation/network_integration/advertiser_campaigns/index.rst:543]: I think that's probably fine.  I'll see what I can do.

**Review by danielmx-dev — APPROVED** [review]
LGTM!

**Comment by nburwell** [comment]
I just found this PR - should this be merged? Was approved and looks low risk

**Comment by NickLindeberg** [comment]
Going to pass on adding Daniels suggestion for scope reasons

**Review by oleksiy — APPROVED** [review]
verified the docs render with @NickLindeberg. Merging the PR shortly into 2022-08-01 branch

**Pull request merged**